schemahcl/extension.go

```diff
@@ -376,11 +376,12 @@ func setPtr(field reflect.Value, cv cty.Value) error {
 // setSliceAttr sets the value of attr to the slice field. This function expects both the target field
 // and the source attr to be slices.
 func setSliceAttr(field reflect.Value, attr *Attr) error {
-	if !attr.V.Type().IsListType() && !attr.V.Type().IsTupleType() {
-		return fmt.Errorf("schemahcl: field is of type slice but attr %q is type: %s", attr.K, attr.V.Type().FriendlyName())
```

Comment:
This is the only change I am concerned about - to me it seems logical this accepts either a value or an iterable, and if it's a single value it is converted into an iterable of one.  However it is a major change!